--- pytest-xdist-1.15.0/testing/acceptance_test.py.orig	2016-08-12 16:40:55.000000000 +0200
+++ pytest-xdist-1.15.0/testing/acceptance_test.py	2017-03-12 09:22:03.916096441 +0100
@@ -30,7 +30,7 @@
         result = testdir.runpytest(p1, "-n1")
         assert result.ret == 1
         result.stdout.fnmatch_lines([
-            "E   ImportError: *__import_of_missing_module*",
+            "E   *Error: *__import_of_missing_module*",
         ])
 
     def test_n2_import_error(self, testdir):
@@ -64,7 +64,7 @@
         result = testdir.runpytest(p1, '--tx=popen', '--tx=popen')
         assert result.ret == 1
         result.stdout.fnmatch_lines([
-            "E   ImportError: *__import_of_missing_module*",
+            "E   *Error: *__import_of_missing_module*",
         ])
 
     def test_manytests_to_one_popen(self, testdir):
